components/shared_user_indicator.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {Tooltip} from 'react-bootstrap';
+import {FormattedMessage} from 'react-intl';
+
+import OverlayTrigger from 'components/overlay_trigger';
+import {Constants} from 'utils/constants';
+
+type Props = {
+    className?: string;
+    withTooltip?: boolean;
+};
+
+const SharedUserIndicator: React.FC<Props> = (props: Props): JSX.Element => {
+    const sharedIcon = (<i className={`${props.className || ''} icon-circle-multiple-outline`}/>);
```

Comment:
FYI, the classNames utility function is nice for things like this since it auto concatenates and handles conditional values nicely
```jsx
import classNames from 'classnames';

<i className={classNames('icon', props.className, {[props.enabled]: 'icon--enabled'})}/>;
```